ejp_xml_pipeline/transform_zip_xml/ejp_manuscript_xml.py

```diff
@@ -425,6 +425,9 @@ def email_node_to_dict(email_node: Element) -> dict:
 def derive_version_id_from_manuscript_id_and_created_timestamp(
         manuscript_id: str,
         created_timestamp: str) -> str:
+    if not created_timestamp:
+        return 'NotAcceptable %s/%s' % (manuscript_id, created_timestamp)
```

Comment:
"This is not acceptable!" - sounds very British :D

Reply:
:D